cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -408,9 +487,9 @@ def test_cluster_param_from_file(mocker, param_key, param_value, expected_value,
         ("base_os", None, None, "Configuration parameter 'base_os' must have a value"),
     ],
 )
-def test_cluster_param_from_file_with_validation(mocker, param_key, param_value, expected_value, expected_message):
+def test_sit_cluster_param_from_file_with_validation(mocker, param_key, param_value, expected_value, expected_message):
```

Comment:
see preiovus comment, we should have test_hit_cluster_param_from_file_with_validation

Reply:
What is the purpose of this test? Looks like it's only testing required parameters, which are the same for HIT and SIT cluster sections.

Reply:
You right